ssl/ssl_lib.c

```diff
@@ -5424,6 +5424,40 @@ int SSL_client_hello_get1_extensions_present(SSL *s, int **out, size_t *outlen)
     return 0;
 }
 
+int SSL_client_hello_get_raw_extensions(SSL *s, unsigned short *exts, size_t *num_exts)
```

Comment:
I think we should use `uint16_t *` rather than `unsigned short *`.  That's a more accurate representation of what we want (the registered values are explicitly 16-bit values), and we would not need a cast on line 5454 below.

Reply:
done.